contracts/src/v0.8/dev/VRFV2Wrapper.sol

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "../vendor/SafeMathChainlink.sol";
+
+import "../ConfirmedOwner.sol";
+import "../interfaces/TypeAndVersionInterface.sol";
+import "../VRFConsumerBaseV2.sol";
+import "../interfaces/LinkTokenInterface.sol";
+import "../interfaces/AggregatorV3Interface.sol";
+import "../interfaces/VRFCoordinatorV2Interface.sol";
+import "../vendor/SafeMathChainlink.sol";
+import "./VRFV2WrapperInterface.sol";
+
+/**
+ * @notice A wrapper for VRFCoordinatorV2 that provides an interface better suited to one-off
+ * @notice requests for randomness.
+ */
+contract VRFV2Wrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsumerBaseV2, VRFV2WrapperInterface {
+  using SafeMathChainlink for uint256;
+
+  LinkTokenInterface public immutable LINK;
+  AggregatorV3Interface public immutable LINK_ETH_FEED;
+  ExtendedVRFCoordinatorV2Interface public immutable COORDINATOR;
+  uint64 public immutable SUBSCRIPTION_ID;
```

Comment:
Do we foresee a situation where we'd might have to mutate this? I can't think of any off the top of my head, besides somehow the subscription owner being compromised (though if the owner is a Gnosis safe, should be OK?)

Reply:
Yup! That's why I think it's safe to make this immutable, it should be 100% impossible to change anything about the subscription since the wrapper doesn't expose any way to do that